js/templates/modals/purchase/purchase.html

```diff
@@ -144,7 +144,7 @@
             <div class="flexVCentClearMarg">
               <h2 class="h4 flexExpand required"><%= ob.polyT('purchase.paymentTypeTitle') %></h2>
               <% if (ob.showModerators) { %>
-                <input type="checkbox" id="purchaseVerifiedOnly" class="js-purchaseVerifiedOnly"<% if(ob.showOnlyVerified) print('checked') %>>
+                <input type="checkbox" id="purchaseVerifiedOnly" class="js-purchaseVerifiedOnly"<% if(ob.showOnlyVerified && !ob.unverifedSelected) print('checked') %>>
```

Comment:
What is the difference between `showOnlyVerified`  and `unverifedSelected`?

It seems like we may have 2 sources of truth on that state or maybe I'm misunderstanding the distinction.

Reply:
unverifiedSelected mean you have selected an unverified moderator, so the ability to show only verified moderators gets disabled, otherwise you end up hiding the selected moderator, and the UX gets weird.

It'd probably be better to do something in the view to set things up so showOnlyVerified can be true or false as long as unverifiedSelected is false, but showOnlyVerified can only be false if unverifiedSelected is true.

I was being lazy with this fix.